flink/v1.19/flink/src/main/java/org/apache/iceberg/flink/sink/BaseDeltaTaskWriter.java

```diff
@@ -38,7 +38,7 @@
 import org.apache.iceberg.relocated.com.google.common.collect.Sets;
 import org.apache.iceberg.types.TypeUtil;
 
-abstract class BaseDeltaTaskWriter extends BaseTaskWriter<RowData> {
+public abstract class BaseDeltaTaskWriter extends BaseTaskWriter<RowData> {
```

Comment:
In all of these cases where we need to expose the class, we might want to add the `@Internal` annotation to highlight that these are internal classes